install/empty_data.php

```diff
@@ -34,3994 +34,4026 @@
 use Glpi\Socket;
 use Glpi\Toolbox\Sanitizer;
 
-$tables = [];
-
-$tables['glpi_apiclients'] = [
-    [
-        'id'               => 1,
-        'entities_id'      => 0,
-        'is_recursive'     => 1,
-        'name'             => 'full access from localhost',
-        'is_active'        => 1,
-        'ipv4_range_start' => "2130706433", //value from MySQL INET_ATON('127.0.0.1')
-        'ipv4_range_end'   => "2130706433", //value from MySQL INET_ATON('127.0.0.1')
-        'ipv6'             => '::1',
-    ],
-];
-
-foreach (Blacklist::getDefaults() as $type => $values) {
-    foreach ($values as $value) {
-        $tables['glpi_blacklists'][] = [
-            'type' => $type,
-            'name'    => $value['name'],
-            'value'   => $value['value'],
+// Use anonymous class so we can have constants that define special values without polluting the global table
+// and adding unnecessary variables to IDE autocomplete data that may result in errors
+$empty_data_builder = new class
+{
+    /** @var int Self-service profile ID */
+    const PROFILE_SELF_SERVICE = 1;
+    /** @var int Observer profile ID */
+    const PROFILE_OBSERVER     = 2;
+    /** @var int Admin profile ID */
+    const PROFILE_ADMIN        = 3;
+    /** @var int Super-admin profile ID */
+    const PROFILE_SUPER_ADMIN  = 4;
+    /** @var int Hotliner profile ID */
+    const PROFILE_HOTLINER     = 5;
+    /** @var int Technician profile ID */
+    const PROFILE_TECHNICIAN   = 6;
+    /** @var int Supervisor profile ID */
+    const PROFILE_SUPERVISOR   = 7;
+    /** @var int Read-only profile ID */
+    const PROFILE_READ_ONLY    = 8;
+
+    const USER_GLPI            = 2;
+    const USER_POST_ONLY       = 3;
+    const USER_TECH            = 4;
+    const USER_NORMAL          = 5;
+    const USER_SYSTEM          = 6;
```

Comment:
You should use these constants to force `glpi_users` and `glpi_profiles` `id` values.